ui/app/components/license-info.js

```diff
@@ -0,0 +1,32 @@
+import Component from '@ember/component';
+import { allFeatures } from 'vault/helpers/all-features';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  expirationTime: '',
+  startTime: '',
+  licenseId: '',
+  features: null,
+  text: '',
+  showForm: false,
+  isTemporary: computed('licenseId', function() {
+    return this.get('licenseId') === 'temporary';
```

Comment:
One of the update with 3.4 is that we don't have to do `this.get('attribute')` anymore  (unless `unkownProperty` is being used but I'm not sure that's anywhere in the Vault code) - now you can do `this.attribute`